cmd/hatchet-migrate/migrate/migrations/20250428175026_v1_0_17.sql

```diff
@@ -0,0 +1,34 @@
+-- +goose Up
+-- +goose StatementBegin
+CREATE TABLE v1_events_olap (
+    tenant_id UUID NOT NULL,
+    id BIGINT GENERATED ALWAYS AS IDENTITY,
+    inserted_at TIMESTAMPTZ NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    generated_at TIMESTAMPTZ NOT NULL,
```

Comment:
Is this referring to the original event timestamp?

Reply:
yeah I was thinking we'd want a timestamp corresponding to the time the event was pushed, not sure if we should consider that the "insert time" though